src/main/java/no/unit/nva/doi/DataciteClient.java

```diff
@@ -8,21 +8,22 @@
 
 public class DataciteClient {
 
-    private static final String dataciteBaseUrlString = "https://data.datacite.org";
+    public static final String DATACITE_BASE_URL_STRING = "https://data.datacite.org";
```

Comment:
We should pass the URIs in as environment variables

Reply:
I suggest to create an issue for this.

Reply:
https://unit.atlassian.net/browse/NP-690